terraform-infra-approvals/global.json

```diff
@@ -64,6 +64,7 @@
     {"source":  "git@github.com:hmcts/terraform-module-postgresql-flexible?ref=postgres-vnet-provider"},
     {"source":  "git@github.com:hmcts/cnp-module-postgres?ref=redundant-backup-optional"},
     {"source":  "git@github.com:hmcts/terraform-module-sdp-db-user?ref=master"},
-    {"source":  "git@github.com:hmcts/terraform-module-application-insights?ref=fix-count"}
+    {"source":  "git@github.com:hmcts/terraform-module-application-insights?ref=fix-count"},
+    {"source":  "git@github.com:hmcts/terraform-module-postgresql-flexible?ref=CME-120"}
```

Comment:
Why run of a branch instead of merging into `main`?

Reply:
we want to test the things before merging into master